src/tests/dbus-tests/test_20_LVM.py

```diff
@@ -1,32 +1,33 @@
-import unittest
-import storagedtestcase
 import dbus
 import os
-import subprocess
 import time
 
+import storagedtestcase
+import utils
```

Comment:
Not a strong opinion about this one: but wouldn't it make sense to merge these two? The methods provided by utils.py are pretty useful for almost every test case. So perhaps teaching storagedtestcase to do file operations and execute commands would make sense.